packages/jaeger-ui/src/components/TraceDiff/TraceDiffGraph/TraceDiffGraph.css

```diff
@@ -42,6 +42,7 @@ limitations under the License.
 .TraceDiffGraph--dag {
   transition: background 0.5s ease;
   stroke-width: 0.9;
+  z-index: -1;
```

Comment:
I understand your intent, but I don't understand how -1 achieves that.

Reply:
First, is this the popover that you're talking about?
![image](https://github.com/jaegertracing/jaeger-ui/assets/3523016/fef7a035-ddf8-4a79-a172-c43739bc19a6)

It opens when clicking on the header, and closes when clicking on the header again. I agree it would be good if it closed when clicking anywhere else - with your change it does seem to happen partially, i.e. when I click on [vs] column or on the menu bar. But it doesn't work if I click on the graph.

Reply:
With your change I am seeing weird behavior when clicking out of the component - it starts disappearing, then flushes before completely going away.

Reply:
> First, is this the popover that you're talking about?
> ![image](https://github.com/jaegertracing/jaeger-ui/assets/3523016/fef7a035-ddf8-4a79-a172-c43739bc19a6)
> 

Yes

Reply:
> With your change I am seeing weird behavior when clicking out of the component - it starts disappearing, then flushes before completely going away.

Oh that happens without my change as well, Probably some problem with the popover. I tried fixing it by giving some transition but nothing seems to work.

Reply:
you're right...